packages/mongo-nav/src/Loading.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { LogoMark } from '@leafygreen-ui/logo';
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const orgNavContainer = css`
+  height: 60px;
+  width: 100%;
```

Comment:
We shouldn't need this with `display: flex;`

Reply:
👍